nestkernel/clopath_archiving_node.cpp

```diff
@@ -64,13 +64,15 @@ nest::ClopathArchivingNode::ClopathArchivingNode( const ClopathArchivingNode& n
 void
 nest::ClopathArchivingNode::init_clopath_buffers()
 {
-  // Implementation of the delay of the convolved membrane potentials. This
-  // delay is not described in Clopath et al. 2010 but is present in the code
-  // (https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=144566) on
-  // ModelDB which was presumably used to create the figures in the paper.
-  // Since we write into the buffer before we read from it, we have to
-  // add 1 to the size of the buffers.
-  delayed_u_bars_idx_ = 0;
+  /*
+   * Implementation of the delay of the convolved membrane potentials. This
+   * delay is not described in Clopath et al. 2010 but is present in the code
+   * (https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=144566) on
+   * ModelDB which was presumably used to create the figures in the paper.
+   * Since we write into the buffer before we read from it, we have to
+   * add 1 to the size of the buffers.
+   */
+  layed_u_bars_idx_ = 0;
```

Comment:
Here's another one for fixing the build
```suggestion
  delayed_u_bars_idx_ = 0;
```